src/Sentry/Internal/Hub.cs

```diff
@@ -56,6 +56,12 @@ public Hub(SentryOptions options)
             _rootScope = PushScope();
         }
 
+        public IHub Clone()
+        {
+            _options.DiagnosticLogger?.LogDebug("Attempt to clone a Hub");
+            return new Hub(_options);
```

Comment:
Is this a `TODO`? The Hub isnt' being cloned here, it's just creating a new one with the same options

Reply:
https://github.com/getsentry/sentry-android/blob/master/sentry-core/src/main/java/io/sentry/core/Hub.java#L589

it's what the java code does, I didn't add the rest because there are no stacks from what I looked.

Reply:
The stack is in the ScopeManager. That's where the state (tags, etc) are kept and Clone needs to copy them over to the new Hub